src/test/scala/io/iohk/ethereum/domain/BlockHeaderSpec.scala

```diff
@@ -6,14 +6,15 @@ import org.scalatest.prop.PropertyChecks
 import org.spongycastle.util.encoders.Hex
 
 class BlockHeaderSpec extends FunSuite with PropertyChecks  {
+  // FIXME: Change hashes with the version with slotNumber
```

Comment:
Is this FIXME still pending? AFAIK this was addressed in this PR

Reply:
good catch hehehe